docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,484 @@
+  Expect this metrics to be `0` when everything is currently deployed.
+  It will occasionally rise to `n` during deployments and then fall back to `0` when deployments are completed.
+
+There are many other Keptn metrics.
+
+## Make DORA metrics more user friendly
+
+It is much more user friendly to provide dashboards for metrics, logs and traces.
+So let's install new Observability components to help us:
+
+- Cert manager: Jaeger requires cert-manager
+- Jaeger: Store and view DORA deployment traces
+- Prometheus: Store DORA metrics
+- OpenTelemetry collector: Scrape metrics from the above DORA metrics endpoint & forward to Prometheus
+- Grafana (and some prebuilt dashboards): Visualise the data
+
+![add observability](../assets/install01.png)
+
+## Install Cert Manager
+
+Jaeger requires Cert Manager, so install it now:
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.2/cert-manager.crds.yaml
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager --namespace cert-manager --version v1.12.2 jetstack/cert-manager --create-namespace --wait
+```
+
+## Install Jaeger
+
+Save this file as `jaeger.yaml` (it can be saved anywhere on your computer):
+
+```shell
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+spec:
+  strategy: allInOne
+```
+
+Install Jaeger to store and visualise the deployment traces generated by Keptn:
+
+```shell
+kubectl create namespace observability
+kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/v1.46.0/jaeger-operator.yaml -n observability
+kubectl wait --for=condition=available deployment/jaeger-operator -n observability --timeout=300s
+kubectl apply -f jaeger.yaml -n keptn-lifecycle-toolkit-system
+kubectl wait --for=condition=available deployment/jaeger -n keptn-lifecycle-toolkit-system --timeout=300s
+```
+
+Port-forward to access Jaeger:
+
+```shell
+kubectl -n keptn-lifecycle-toolkit-system port-forward svc/jaeger-query 16686
+```
+
+Jaeger is available on `http://localhost:16686`
+
+## Install Grafana dashboards
+
+Create some Keptn Grafana dashboards that will be available when Grafana is installed and started:
+
+```shell
+kubectl create ns monitoring
+kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/main/examples/support/observability/config/prometheus/grafana-config.yaml
+kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/main/examples/support/observability/config/prometheus/grafana-dashboard-keptn-applications.yaml
+kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/main/examples/support/observability/config/prometheus/grafana-dashboard-keptn-overview.yaml
+kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/main/examples/support/observability/config/prometheus/grafana-dashboard-keptn-workloads.yaml
+```
+
+## Install Grafana datasources
```

Comment:
Those links should point to the latest released versions of the files and not to main. Then they can be updated with release-please during release of KLT.
Something like: (please look at the raw code of this comment, the release please annotation comments are hidden in the preview)

<!---x-release-please-start-version-->
```shell
kubectl create ns monitoring
kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/klt-v0.8.1/examples/support/observability/config/prometheus/grafana-config.yaml
kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/klt-v0.8.1/examples/support/observability/config/prometheus/grafana-dashboard-keptn-applications.yaml
kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/klt-v0.8.1/examples/support/observability/config/prometheus/grafana-dashboard-keptn-overview.yaml
kubectl apply -f https://raw.githubusercontent.com/keptn/lifecycle-toolkit/klt-v0.8.1/examples/support/observability/config/prometheus/grafana-dashboard-keptn-workloads.yaml
```
<!---x-release-please-end-->